web-app-router/scenes/Portal/Teams/TeamId/Team/page/Apps/App/index.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
```

Comment:
lets write this such that only verified apps will have logos that display. Then you can just construct the url using the `getCDNImageUrl(app_id, path)`. 

Unfortunately since we don't know the path it will be something like 10393.png we need to fetch DB. but just filter on verified apps since we would have to get signedurls to display unverified ones